src/Logger-DeviceClass_template/main.py

```diff
@@ -225,47 +225,47 @@ def initialize(self):
 
     def deinitialize(self):
         # called only once at the end of the measurement
-        debug("deinitialize")
-
-    def poweron(self):
-        # called if the measurement procedure enters a branch of the sequencer
-        # and the module has not been used in the previous branch
-        debug("poweron")
-
-    def poweroff(self):
-        # called if the measurement procedure leaves a branch of the sequencer
-        # and the module is not used in the next branch
-        debug("poweroff")
+        debug("-> deinitialize")
 
     def configure(self):
         # called if the measurement procedure enters a branch of the sequencer
         # and the module has not been used in the previous branch
-        debug("configure")
+        debug("->  configure")
 
     def unconfigure(self):
         # called if the measurement procedure leaves a branch of the sequencer
         # and the module is not used in the next branch
-        debug("unconfigure")
+        debug("->  unconfigure")
 
     def signin(self):
         # called if the variation of the module that is loading this device class starts
-        debug("signin")
+        debug("->   signin")
 
     def signout(self):
         # called if the variation of the module that is loading this device class ends
-        debug("signout")
-
+        debug("->   signout")
+        
     def reconfigure(self, parameters, keys):
         """'reconfigure' is called whenever parameters of the GUI change by using the {...}-parameter system."""
         debug()
-        debug("reconfigure")
-        debug("Parameters:", parameters)
-        debug("Changed keys:", keys)
+        debug("->  reconfigure")
+        debug("->  Parameters:", parameters)
+        debug("->  Changed keys:", keys)
 
         ### The following two lines are the default behavior that is used by EmptyDevice
         ### if you do not override 'reconfigure'
         # self.get_GUIparameter(parameters)
         # self.configure()
+        
+    def poweron(self):
```

Comment:
Because poweron is called after signin at the start, it is better for visualization to give a poweron another level of indentation
![image](https://github.com/SweepMe/instrument-drivers/assets/61842286/fb5e7347-a658-4df4-8d4e-99b2a79f4059)
This way one can easily see that if the indentation gradually increases everything is ok.

Reply:
Changed it to a new scheme

![image](https://github.com/SweepMe/instrument-drivers/assets/61842286/20d324cc-7524-4dfd-96ae-6ade3e5e9b2d)

connect, initialize, deinitialize, disconnect -> no indent
configure, reconfigure, signin, poweron, poweroff, signout, unconfigure -> 2 spaces indent
alle point wise function -> 4 spaces indent